ui/components/app/multilayer-fee-message/multi-layer-fee-message.js

```diff
@@ -21,19 +21,21 @@ export default function MultilayerFeeMessage({
   const [fetchedLayer1Total, setLayer1Total] = useState(null);
 
   let layer1Total = 'unknown';
```

Comment:
Any reason we're not using null or undefined here instead of the string unknown?

Reply:
`Unknown` is string displayed to the user is L1 gas estimates are not available:

![Screenshot 2023-04-10 at 12 57 54 PM](https://user-images.githubusercontent.com/2182307/230850587-25ddea61-eb07-4cc2-abf6-6c61678cd16e.png)

I improved it to use internationalisation.